docs/getting-started-with-cosmosdb.md

```diff
@@ -0,0 +1,103 @@
+## Getting Started with Scalar DB and Cosmos DB
+
+## Overview
+This document briefly explains how you can get started a simple electronic money application with scalar DB and Cosmos DB.
+
+## Install prerequisites
+
+Scalar DB is written in Java and uses Cosmos DB as an underlining storage implementation, so the following software is required to run it.
```

Comment:
`so` is not logically connecting the two sentences.
uses Cosmos DB => (so) => JDK, libraries are needed ???

Reply:
Sorry for the inconvenience.
Document updated.

> uses Cosmos DB => (so) => JDK, libraries are needed ???

Yes
Because scalardb and sample application are written in java.